### ISSUE TITLE
M1 Mac running Monterey issues (multiple)

### ISSUE THREAD

**Issue by rtbarber** [mcomas/codapack#17]
I am unable to get the program running on an M1 Mac Running Monterey.  I am running codapack 2.03.01

Problems include:
1) Data > Create new table allows me to specify the size of the table, but no amount of clicks in the boxes will let me enter any data at all
2) File > Open Workspace or Add Workspace - the finder doesn't see the file in the folder.  Note that it doesn't see anything, even after choosing all files.  In many cases I was unable to navigate because it stops showing me any further directories.  I moved all the files to the desktop and can GET there but cannot see any files at all, including the files I moved there to try to access.  Also tried documents.  Certainly couldn't get the files out of my dropbox folder, as I couldn't navigate into the structure past 1 level.
3) File > Import Data - Same issues with navigation and not seeing files.  
4) Doubleclick on cdp file - I associate Codapack with it and the program starts, but the file doesn't open.

In short, I cannot get any data IN to the program to try to manipulate in any way.  Any ideas?
![2023-07-24_20-08-34](https://github.com/mcomas/codapack/assets/3385547/6c14a2a0-6796-4b9b-9e63-1ca6164459b9)
